compass/landice/tests/ismip6_run/ismip6_ais_proj2300/set_up_experiment.py

```diff
@@ -268,5 +310,95 @@ def run(self):
         """
         Run this step of the test case
         """
+
+        config = self.config
+        logger = self.logger
+        section = config['ismip6_run_ais_2300']
+        sea_level_model = section.getboolean('sea_level_model')
+        if sea_level_model:
+            self._build_mapping_files(config, logger)
+
         run_model(step=self, namelist='namelist.landice',
                   streams='streams.landice')
+
+    def _build_mapping_files(self, config, logger):
+        """
+        Build mapping files between the MALI mesh and the SLM grid.
+
+        Parameters
+        ----------
+        config : compass.config.CompassConfigParser
+            Configuration options for a ismip6 forcing test case
+
+        logger : logging.Logger
+            A logger for output from the step
+        """
+
+        config = self.config
+        section = config['ismip6_run_ais_2300']
+        init_cond_path = section.get('init_cond_path')
+        nglv = section.getint('nglv')
+
+        # create the mapping files
+        # first create scrip files
+        mali_scripfile = 'mali_scripfile.nc'
+        slm_scripfile = f'slm_nglv{nglv}scripfile.nc'
+        mali_meshfile = 'mali_meshfile.nc'
```

Comment:
Let's change this to have a more descriptive filename, e.g.
```suggestion
        mali_meshfile = 'mali_meshfile_sphereLatLon.nc'
```
I know that this file gets deleted when this function completes, but that way if an error occurs, this file isn't confusing.